es6-shim/es6-shim.d.ts

```diff
@@ -503,7 +503,7 @@ interface PromiseConstructor {
      * a resolve callback used resolve the promise with a value or the result of another promise,
      * and a reject callback used to reject the promise with a provided reason or error.
      */
-    new <T>(executor: (resolve: (value?: T | PromiseLike<T>) => void, reject: (reason?: any) => void) => void): Promise<T>;
+    new <T>(executor: (resolve: (value?: T | PromiseLike<T>) => void, reject: (reason?: any) => any) => void): Promise<T>;
```

Comment:
Why would you change this here? The `reject` callback has provided to the `executor` has no return value, so `void` is correct here.